src/Log/Helpers/LegacyLogsTable.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Give\Log\Helpers;
+
+/**
+ * Class LogsLegacyTable
+ * @package Give\Log\Helpers
+ *
+ * @since @unreleased
+ */
+class LegacyLogsTable {
+	/**
+	 * Check if legacy logs table exists
+	 *
+	 * @return bool
+	 */
+	public function exist() {
```

Comment:
I can see that this function can be `static`.

cc: @JasonTheAdams

Reply:
Nah, leave it as an instance. This is acting as a repository, so using it via dependency-injection is preferred. This way the classes using this can mock it in tests and not have to seed the database.

Reply:
👍

Reply:
> This function can be `static`.

Can be private as well.